Sources/Fullscreen/FullscreenGallery/FullscreenGalleryOverlayView.swift

```diff
@@ -40,7 +40,8 @@ class FullscreenGalleryOverlayView: UIView {
 
     // MARK: - Private properties
 
-    private static let dismissButtonSize: CGFloat = 30.0
+    private static let dismissButtonTappableSize: CGFloat = 55.0
+    private static let dismissButtonVisualSize: CGFloat = 30.0
```

Comment:
Are you using this anywhere?

Reply:
Snap, no – remnant from an earlier revision. Removing!